### DISCUSSION TITLE
RiMusic is #63 most downloaded app on f-droid

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1933] [General]
A recent data came out from divest os team and to my surprise RiMusic was on 63rd place. This is only counting f-droid downloads not izzy otherwise it'll probably be top 50. A very good achievement i would say in such a short period of time. Congrats.

https://divestos.org/pages/fdroid_stats

**Comment by ikanakova** [comment]
Congrats @fast4x! 🎉 But I'm not surprised - RiMusic is a great app.

**Comment by fast4x** [comment]
WOW AMAZING! Thanks to All!!!!!!!!!!!!!

**Comment by fast4x** [comment]
also in home... wow
![Screenshot_2024-05-19-11-44-30-914_com.microsoft.emmx.jpg](https://github.com/fast4x/RiMusic/assets/1484476/f4bcae1a-ed71-4215-ba31-b75311d43f55)

**Reply by NikunjK8** [reply]
Divest os is a security focused os. I guess RiMusic getting updates every week is probably the reason they recommend it because it minimizes any chance of vulnerability.

**Reply by fast4x** [reply]
I'm honoured

**Reply by ikanakova** [reply]
I think it's more because RiMusic doesn't require any unnecessary permissions and doesn't depend on any tracking or non-free services besides YTM.
Because for example Transistor is updated exceptionally (at least against RiMusic 😁) and is listed as such.

**Reply by fast4x** [reply]
Yes, @ikanakova is logical to think that the reason is related to the security level recognized by RiMusic since Divest is focused on security. Great, it’s a good indicator of the quality of RiMusic. This at least for me is a great reason to continue development in the right way.

**Reply by NikunjK8** [reply]
@ikanakova Honestly permission part can be said for every ytm client. I've tried dozen of them in the past. All of them also happen to be open source. The only closed source one i remember is ymusic.io. Either way RiMusic is still the best one.

**Reply by NikunjK8** [reply]
@ikanakova Btw if you're free can you take a look at these? #1395 #1877

**Comment by fast4x** [comment]
Wow RiMusic vides on YouTube 

https://youtu.be/NoXIII2c_iw?feature=shared

https://youtu.be/4ekWlfcDxSw?feature=shared

https://youtu.be/FEIncPYl99s?feature=shared

**Reply by NikunjK8** [reply]
One has 134k views. Crazyyyy

**Comment by rsssxd** [comment]
When will the latest version hit the accrescent store?
@fast4x

**Reply by fast4x** [reply]
Hi, today.

**Reply by rsssxd** [reply]
Thanks.
